src/Microsoft.Health.Fhir.CosmosDb/Features/Search/FhirCosmosSearchService.cs

```diff
@@ -59,7 +72,8 @@ public class FhirCosmosSearchService : SearchService
         private async Task<SearchResult> ExecuteSearchAsync(
```

Comment:
Instead of adding a boolean, I think we should just split the method; it's not sharing much logic anyways. Perhaps having `ExecutSearchAsync` and `ExecuteCountSearchAsync` or something like that?

Reply:
Same as below, I created [#AB70828](https://microsofthealth.visualstudio.com/Health/_workitems/edit/70828).